src/CropManager.sol

```diff
@@ -107,10 +107,10 @@ contract CropManagerImp {
         }
     }
 
-    function join(address crop, address urn, uint256 val) external {
+    function join(address crop, address usr, uint256 val) external {
         TokenLike(CropLike(crop).gem()).transferFrom(msg.sender, address(this), val);
         TokenLike(CropLike(crop).gem()).approve(crop, val);
-        CropLike(crop).join(getOrCreateProxy(urn), urn, val);
+        CropLike(crop).join(getOrCreateProxy(usr), usr, val);
```

Comment:
Yeah the usage of `urn` here was not so much a mistake, but it came from the original language in the `crop` which specified the destination urn to deposit the gems which could be different than the source user, but I think I agree with this change. It's less confusing.

Reply:
Yeah I had that doubt, but also felt now makes more sense to call it `usr`, as the `urn` is actually the `proxy`.